.jenkins/build_container.JenkinsFile

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/groovy
+def LABEL = "build-jenkins-casc-" + UUID.randomUUID().toString().take(8)
+pipeline {
+    parameters {
+        choice(name: 'TAG', choices: ['git-commit','git-tag'],  description: 'Which info to use in container tag?')
+    }
+    options {
+      timestamps()
+      ansiColor('xterm')
+    }
+    agent {
+      kubernetes {
+        label LABEL
+        yamlFile '.jenkins/pods/build_container.yaml'
+      }
+    }
+    stages {
+      stage('Container') {
+        steps {
+          if (params.TAG != '') {
```

Comment:
this will not work, as you're here in the declarative (non-superpowers) context. This plain groovy would just work outside of `pipeline` or inside of `script`. The declarative pipeline equivalent would be something (untested) like 

```java
when {
  not {
    equals expected: '', actual: params.tag
  }
}
```
or
```java
when {
  expression {
    params.TAG != ''
  }
}
```

Reply:
So, let's give some superpowers :D